packages/showcase/package.json

```diff
@@ -31,5 +31,26 @@
   "volta": {
     "node": "10.20.1",
     "yarn": "1.22.4"
+  },
+  "babel": {
+    "presets": [
+      "es2015",
+      "stage-0",
+      "react"
+    ],
+    "plugins": [
+      [
+        "module-resolver",
+        {
+          "root": [
+            "."
+          ],
+          "alias": {
+            "react-vis/*": "../react-vis/src/*",
```

Comment:
Interesting that we need this statement. Makes sense.
Do we need to remove some duplication from the storybook webpack config?

Reply:
We need this so that the tests will pick up the same instances of the components as the tests.  
Eventually we will need the tests to use their own plots.
The way things are now, a simple change to the showcase can break the tests, which definitely isn't ideal.